README.md

```diff
@@ -4,26 +4,48 @@ Elastic Stack sample setup for Nginx with Filebeat using Docker.
 
 ## Server:  Elasticsearch, Logstash & Kibana
 
-### Build
+### Using docker-compose
+
+> You must install docker-compose first: [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)
+
+- Move to the project
+```
+cd ./elastic-stack
+```
+
+- Set environment variables
+```
+cp .env.template .env
```

Comment:
We should precise that we need to edit the file with our wanted ports and S3 credentials.